templates/ue5_build.Jenkinsfile

```diff
@@ -0,0 +1,76 @@
+// Fetch Shared Library from Github
+library identifier: 'JenkinsSharedLib@master',
+    retriever: modernSCM([
+      $class: 'GitSCMSource',
+      credentialsId: '', // Public repo, no credentials needed
+      remote: 'https://github.com/jezzye13/JenkinsSharedLib'
```

Comment:
remote url should be "https://github.com/DavidtKate/JenkinsSharedLib" to fit appropriate repo which template is stored in

Reply:
Ja dat is logisch ik wist niet zeker hoe snel jij zou reageren op mijn pull request.